test/e2e/aws_test.go

```diff
@@ -679,6 +679,27 @@ func TestAWSChangeS3Encryption(t *testing.T) {
 			t.Errorf("unable to find environment variable: wanted %s", val.Name)
 		}
 	}
+
+	// If a user sets Encrypt: false, don't overwrite it
```

Comment:
That still changes the behaviour of the operator. In the current version a customer can provide only `keyID` or `cloudFront`, the operator will populate everything else and set `encrypt` to `true`.

I don't think we can fix this problem without breaking backward compatibility and without introducing a new API field.

Reply:
An idea, can't we make the encrypt field a pointer to a bool? At least I have seen this approach being used to differentiate between "user has set it to false" from "user did not set it". My two cents.

Reply:
No, it's a bad idea to have different logic for `null` and `false`. We've already discussed it,  @deads2k suggested to deprecate the existing field and add `insecureDisableEncryptedStorage`.

Reply:
Makes sense.